### ISSUE TITLE
Some buttons do not work

### ISSUE THREAD

**Issue by PropGit** [parallaxinc/propside#287]
From a customer report:

> All of the buttons in simpleide ver 1.1.2 do not work.
> Such as support, print, editor, and sales.

**Comment by AndyLindsay** [comment]
I saw the email where this was reported and am able to reproduce the Print button issue in Windows 10.  The email also mentions editor, sales, and support buttons.  Those have not been a part of the SimpleIDE menus/buttons that I am aware of.  They might have been on an older version, but to my knowledge, all other existing buttons and menu items in the current (1.1.2 version) work.

**Comment by PropGit** [comment]
I agree; I don't remember most of those being in the IDE, but I also confirmed that Print does not work in Windows 7.
